tests/benchmark-runner-tests.mjs

```diff
@@ -112,9 +112,9 @@ describe("BenchmarkRunner", () => {
             let _runSuiteStub, _finalizeStub, _loadFrameStub, _appendFrameStub, _removeFrameStub;
 
             before(async () => {
-                _runSuiteStub = stub(runner, "_runSuite").callsFake(async () => null);
+                _runSuiteStub = stub(SuiteRunner.prototype, "_runSuite").callsFake(async () => null);
```

Comment:
This looks so brittle :D but let's not fix that now

Reply:
lol... yeah, generally not too happy about how many internals we test, but then again I don't have a clear idea on how to fix this.